fontc/src/args.rs

```diff
@@ -63,6 +63,10 @@ pub struct Args {
     // https://github.com/googlefonts/fontmake/blob/6a8b2907/Lib/fontmake/__main__.py#L443
     #[arg(long, default_value = "false")]
     pub keep_direction: bool,
+
+    /// Don't rename glyphs with production names
```

Comment:
Can you add a note that this is setup to match the fontmake flag and a reference to it similar to the one on keep_direction?